recipe/meta.yaml

```diff
@@ -8,11 +8,10 @@ source:
   url:  http://sourceforge.net/projects/pyqt/files/PyQt5/PyQt-5.9.2/PyQt5_gpl-5.9.2.tar.gz
   sha1: 058c5b37981b515cc4e16faa04a4a6d8892cdaa9
   patches:
-    - 0001-Ignore-lack-of-shared-python-lib.patch
-    - 0002-configure.patch
```

Comment:
Why did you remove these patches? I think `0001-Ignore-lack-of-shared-python-lib.patch` conflicts with what you're trying to do here, but `0002-configure.patch` contained a fix for Linux too which you simply ignored without explanation.

Reply:
Both patches were just masking the problem by simply ignoring the fact that configure.py can’t find the shared library like before. The only patch that was doing something was 0001-configure but since it was removed before I did not bother with that.

0001-Ignore... simply removes the error message when the shared library is not found and this leads to issues that were only solved by the fact that 0002-configure is telling the linked to ignore the not found lib for Mac and maybe the same for Linux.

The build went fine and looks like the tests passed.

My patch makes the two patches mentioned above obsolete as it properly points to the python shared library (you can take a look at the build log and confirm it yourself) and with that there is no reason to ignore undefined.

Reply:
>  with that there is no reason to ignore undefined.

There is. The python executable is linked to the static python library and not the shared library. When you link the python shared library to pyqt5 and you load pyqt5 from the python executable, you are bringing in two incompatible copies of python into the same process leading to crashes

Reply:
Do you have a crash to report with this scenario?
For the designer plugin it loads the python library and that is all it
needs.
With the previous patch it just could not find the python symbols that were
undefined and that resulted in a crash.